java7/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:12.04
+ENV DEBIAN_FRONTEND noninteractive
+
+# Accept oracle license in debconf
+ADD java7.debconf /tmp/java7.debconf
+# Add webupdate8 apt repo
+ADD java7.list /etc/apt/sources.list.d/java7.list
+# Add webupdate8 signing key
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 7B2C3B0889BF5709A105D03AC2518248EEA14886
+RUN apt-get update -qq
```

Comment:
like in https://github.com/scrapinghub/devbox/blob/master/base/build-image.sh#L22-L34